src/govuk/components/summary-list/template.njk

```diff
@@ -7,38 +7,82 @@
+            {%- for action in params.actions.items -%}
+              <li class="govuk-summary-card__action">
+                {{- _actionLink(action) -}}
+              </li>
+            {%- endfor -%}
+          </ul>
+        {% endif -%}
+      {%- endif -%}
+    </div>
+
+    <div class="govuk-summary-card__content">
+      {{- caller() -}}
+    </div>
+  </div>
+{% endmacro -%}
+
 {# Determine if we need 2 or 3 columns #}
 {% set anyRowHasActions = false %}
 {% for row in params.rows %}
   {% set anyRowHasActions = true if row.actions.items | length else anyRowHasActions %}
 {% endfor -%}
 
-<dl class="govuk-summary-list {%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-  {% for row in params.rows %}
-    {% if row %}
-      <div class="govuk-summary-list__row {%- if anyRowHasActions and not row.actions.items %} govuk-summary-list__row--no-actions{% endif %} {%- if row.classes %} {{ row.classes }}{% endif %}">
-        <dt class="govuk-summary-list__key {%- if row.key.classes %} {{ row.key.classes }}{% endif %}">
-          {{ row.key.html | safe if row.key.html else row.key.text }}
-        </dt>
-        <dd class="govuk-summary-list__value {%- if row.value.classes %} {{ row.value.classes }}{% endif %}">
-          {{ row.value.html | indent(8) | trim | safe if row.value.html else row.value.text }}
-        </dd>
-        {% if row.actions.items.length %}
-          <dd class="govuk-summary-list__actions {%- if row.actions.classes %} {{ row.actions.classes }}{% endif %}">
-            {% if row.actions.items.length == 1 %}
-              {{ _actionLink(row.actions.items[0]) | indent(12) | trim }}
-            {% else %}
-              <ul class="govuk-summary-list__actions-list">
-                {%- for action in row.actions.items -%}
-                  <li class="govuk-summary-list__actions-list-item">
-                    {{- _actionLink(action) | indent(18) | trim -}}
-                  </li>
-                {%- endfor -%}
-              </ul>
-            {% endif %}
+{%- set summaryList -%}
+  <dl class="govuk-summary-list {%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+    {% for row in params.rows %}
+      {% if row %}
+        <div class="govuk-summary-list__row {%- if anyRowHasActions and not row.actions.items %} govuk-summary-list__row--no-actions{% endif %} {%- if row.classes %} {{ row.classes }}{% endif %}">
+          <dt class="govuk-summary-list__key {%- if row.key.classes %} {{ row.key.classes }}{% endif %}">
+            {{ row.key.html | safe if row.key.html else row.key.text }}
+          </dt>
+          <dd class="govuk-summary-list__value {%- if row.value.classes %} {{ row.value.classes }}{% endif %}">
+            {{ row.value.html | indent(8) | trim | safe if row.value.html else row.value.text }}
```

Comment:
@owenatgov Presume the `indent(8)` need bumping because we've indented another level?

But other lines don't have `indent` so worth a check

Reply:
I completely missed this as something we were doing in the component and spent an interesting hour understanding what it was doing 😂  For anyone who's interested, [here's why we're doing this](https://github.com/alphagov/govuk-frontend/pull/1131/files#r247459722). My rough understanding is that this is to make the source easier to read.

I don't see any reason not to bump this for tidiness.

Reply:
I'm going to double back as I'm debating the value of doing this. We'd need to add some logic to check if the `card` API was populated and add (or not add) the `indent` and it's value based on that. I worry that this is going to add a lot of chuff to the template with not that much reward. What do you think @colinrotherham?

Reply:
Might have to defer to @36degrees wisdom on this one

We do use `indent(n)` quite a lot, see this [comment on the **Button** component](https://github.com/alphagov/govuk-frontend/commit/d8fcb683b327bcd6387f5344994cb9fcbbd8349c#diff-b56c3509e935950b17bc531de661529e2025c837e4e40eceffc165aa4cebd9c8R51):

```njk
{# Indentation is intentional to output HTML nicely #}
{{- iconHtml | safe | trim | indent(2, true) if iconHtml -}}
```

Think I'd want all calls to `indent(n)` to either be correct or removed 😊

Bear in mind we might not see it in action as the first line (of multi-line HTML) isn't indented
https://mozilla.github.io/nunjucks/templating.html#indent

Reply:
I'm doubling back on my double back! I've figured out the intent of these indents: they're for when CSS isn't available. Will action.